ci/travis/determine_tests_to_run.py

```diff
@@ -74,8 +74,11 @@ def get_commit_range():
     RAY_CI_TUNE_AFFECTED = 0
     RAY_CI_SGD_AFFECTED = 0
     RAY_CI_TRAIN_AFFECTED = 0
-    RAY_CI_RLLIB_AFFECTED = 0  # Whether RLlib minimal tests should be run.
-    RAY_CI_RLLIB_FULL_AFFECTED = 0  # Whether full RLlib tests should be run.
+    # Whether only the most important (high-level) RLlib tests should
+    # be run.
+    RAY_CI_RLLIB_AFFECTED = 0
```

Comment:
nit: It's not clear when reading this when a test should be run with this condition. Is there some rule of thumb that we can describe here so that it's clear when this should be used as a condition instead of `RAY_CI_RLLIB_DIRECTLY_AFFECTED`?

Reply:
good point. I think RAY_CI_RLLIB_AFFECTED should focus on high level integration type of tests, since code outside of rllib shouldn't break individual functionalities.
DIRECTLY_AFFECTED should just run everything.

Reply:
Correct, @gjoliver , that's how I changed it:

Where "high-level tests" == RLlib learning tests.